hecuba_py/tests/withcassandra/storagedict_tests.py

```diff
@@ -189,113 +189,6 @@ def set_none_key():
         self.assertRaises(TypeError, set_none_key)
         config.session.execute("DROP TABLE IF EXISTS my_app.somename")
 
-
-
-    def test_paranoid_setitem_nonpersistent(self):
```

Comment:
These tests can be left after removing the line "config.hecuba_type_checking". They check basic features.

Reply:
TD